scope.go

```diff
@@ -382,6 +391,30 @@ func (s *scope) maxQueueTime() time.Duration {
 	return d
 }
 
+type paramsRegistry struct {
+	// key is a hashed concatenation of the params list
+	key uint32
+
+	r map[string]string
```

Comment:
Why not storing `[]config.Param` instead? This way params will be added to proxied requests always in the same order.

Reply:
Indeed, thx for pointing that out!